system_tests/fast_confirm_test.go

```diff
@@ -102,17 +108,35 @@ func TestFastConfirmationWithdrawal(t *testing.T) {
 			)
 			Require(t, err)
 			// Execute the transaction on L1
+			println("balance1: ", builder.L1.GetBalance(t, common.Address{}).Uint64())
 			execTx, err := outboxBinding.ExecuteTransaction(&authL1, outboxProof.Proof, parsedLog.Position, parsedLog.Caller, parsedLog.Destination, parsedLog.ArbBlockNum, parsedLog.EthBlockNum, parsedLog.Timestamp, parsedLog.Callvalue, parsedLog.Data)
 			Require(t, err)
 			execReceipt, err := builder.L1.EnsureTxSucceeded(execTx)
 			Require(t, err)
+			println("balance2: ", builder.L1.GetBalance(t, common.Address{}).Uint64())
```

Comment:
nitpick: same comment about println

Reply:
Added this local debugging log by mistake, removed now.